rust/toolchain.bzl

```diff
@@ -713,13 +714,13 @@ rust_toolchain = rule(
             allow_single_file = True,
             cfg = "exec",
         ),
-        "clippy_driver": attr.label(
-            doc = "The location of the `clippy-driver` binary. Can be a direct source or a filegroup containing one item.",
+        "cargo_clippy": attr.label(
```

Comment:
Is cargo_clippy a real binary?

Reply:
Yep, it is. Or `cargo-clippy` is, to be exact.

If you call `cargo clippy`, `cargo-clippy` is executed. And that uses `clippy-driver` somehow.

Reply:
Wait, so then there is no `cargo_clippy` binary? There is `cargo` and then there is `clippy-driver`. Right?

Reply:
There is a `cargo` binary, a `cargo-clippy` and a `clippy-driver` binary.

The `cargo-clippy` binary is in the same "component" archive as `clippy-driver`.

Reply:
Would it be worth keeping `clippy-driver` around and additionally exposing `cargo-clippy`?

Reply:
I did keep `clippy-driver` around. The diff is misleading. Some lint forced me to sort the attributes alphabetically which makes it appear as if I removed clippy_driver. Instead I just moved it.